webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/PolicyWriterService.scala

```diff
@@ -775,6 +784,13 @@ class PolicyWriterServiceImpl(
     }
   }
 
+  /**
+   * Delete the previous node policies folder
+   */
+  private[this] def deleteNodeFolder(nodeFolder: String): Unit = {
```

Comment:
actually, we could skip it as moveNewNodeFolder does the deletion of dest if it exists